src/engine/source/hlp/src/SpecificParsers.cpp

```diff
@@ -376,3 +378,113 @@ bool parseURL(const char **it, char endToken, URLResult &result) {
 
     return true;
 }
+
+bool parseDomain(const char **it, char endToken, std::vector<std::string> const& captureOpts, DomainResult &result) {
+    static const std::string_view valid_chars {"abcdefghijklmnopqrstuvwxyz"
+                                               "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+                                               "0123456789-_."};
+    static const int DOMAIN_MAX_SIZE = 253;
```

Comment:
could be a `constexpr` instead of a static

Reply:
Changed.